erizo_controller/erizoJS/erizoJSController.js

```diff
@@ -738,6 +732,7 @@ exports.ErizoJSController = (erizoJSId, threadPool, ioThreadPool) => {
       });
     });
     initMetrics();
+    console.log(metrics);
```

Comment:
Is this for your testing?

Reply:
this was for local testing, I should remove it, thanks!